src/ui/Header/Sidebar/AllTabs/Subscriptions.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import SideTag from "../../../SideTag";
+import "./index.css";
+import { ReactComponent as SubscriptionsIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/icons/baricons/subscriptions.svg";
+import { ReactComponent as ActiveSubscriptionsIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/icons/activebaricons/activeSubscriptions.svg";
```

Comment:
this import is incorrect, this is a local file and not within the project folder